app/models/commConns.js

```diff
@@ -19,6 +33,35 @@ class CommConns {
       }).catch(reject);
     })
   }
+
+  static findByCommId (communicationId) {
```

Comment:
this is great

Would make sure to try and not actually use `findManyByAttribute` in real code. Better to define a wrapper method that calls in on the model with a better name. 

So in that test example would prefer that you're calling `CommConns.findManyByCommId` which in turn calls the method you've written above